apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/BuiltInFunctionResourceIntegrationTest.java

```diff
@@ -313,7 +320,112 @@ public void pushNoArray() {
           .then()
           .statusCode(400)
           .body("code", equalTo(400))
-          .body("description", equalTo("The path provided to push to has no array, found {}."));
+          .body(
+              "description",
+              equalTo(
+                  "The path provided to push to has no array, found {\"a\":3,\"b\":[{},{\"nested\":\"value\"},[]]}."));
+    }
+
+    @Test
+    public void setUpdateOneValue() {
+      String setOperation = "{ \"b.[1].nested\": \"newvalue\" }";
```

Comment:
Do nested arrays work? Like `String setOperation = "{ \"b.[1].c.[2].d\": \"newvalue\" }"` ?

Reply:
It should, and I will add a test to that effect :)